src/jit/gentree.h

```diff
@@ -2429,6 +2530,14 @@ struct GenTreeCall final : public GenTree
     }
 #endif // !LEGACY_BACKEND
 
+    // Returns true if the call has retBuf argument
+    bool HasRetBufArg() const { return (gtCallMoreFlags & GTF_CALL_M_RETBUFFARG) != 0; }
+
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+    // Returns true if the call is returning a multi-reg struct value
+    bool HasMultiRegRetVal() const { return varTypeIsStruct(gtType) && !HasRetBufArg(); }
+#endif
+
```

Comment:
I am OK with this for now, but as I mentioned before, I would like to see all of these questions about how types are returned be implemented by the ReturnTypeDesc.